tests/s3_tester.c

```diff
@@ -396,7 +459,7 @@ static bool s_s3_tester_have_meta_requests_finished(void *user_data) {
     AWS_PRECONDITION(user_data);
     struct aws_s3_tester *tester = (struct aws_s3_tester *)user_data;
 
-    return tester->synced_data.meta_requests_finished > 0;
+    return tester->synced_data.meta_requests_finished;
```

Comment:
There's no real change here. I changed the type elsewhere from a "1 bit bool" to an actual bool:
~uint32_t meta_requests_finished : 1;~ -> `bool meta_requests_finished;`

there is no need to optimize for space like this in testing-only structs